module/surveys/helper.js

```diff
@@ -1888,4 +1886,44 @@ module.exports = class SurveysHelper {
         })
     }
 
+
+ /**
+      * Get imported surveys documents.
+      * @method
+      * @name getStartedSurveys
+      * @param  {String} userId - userId of user.
+      * @param  {String} programId - program Id.
+      * @returns {result} - all the survey which user has started in that program. 
+     */
+    static getStartedSurveys(userId, programId){
```

Comment:
@ankitpws this will be moved to usersHelper

Reply:
Moved under usersHelper